modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -1,59 +1,60 @@
-= Policy Level
+= Resource Level Policies
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: policy, custom, ootb, offline, resource level
 
-Policies are by default applied to the entire API, unless configured otherwise. However, you can implement an additional level of granularity for policies, in which access is controlled based on the criteria. These are called _resource-level policies_. 
+Policies are by default applied to the entire API, unless configured otherwise. However, you can implement an additional level of granularity for policies, in which access is controlled based on a criteria. These are called _resource-level policies_. 
 
 At the resource level of granularity, policies are applied to only those requests that match the criteria.  All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
 
 With Mule 4, resource-level policy support now includes HTTP-based APIs along with RAML APIs, as opposed to support for only RAML APIs in Mule 3. A policy applied at a resource level affects all HTTP methods within the resource: PUT, GET, POST, and selected HTTP methods:
 
 image::methods-resources.png[]
 
-You can apply multiple conditions to filter your resources and HTTP methods. The URI template regex can be applied to one, many, or all the methods in your API.
+You can apply multiple conditions to filter your resources and HTTP methods using the xref:configire-uri-template-regex.adoc[URI template regex] to one, multiple, or all the methods in your API.
```

Comment:
Please address tech review comment and apply whatever of this edit is still applicable:
```suggestion
You can apply multiple conditions to filter your resources and HTTP methods by applying the xref:configire-uri-template-regex.adoc[URI template regex] to any number of methods in your API.
```

Reply:
Already fixed. Done! Thanks